package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "pn-email-templates",
+  "version": "1.0.0",
+  "description": "This repo contains the mail templates used by Piattaforma Notifiche, created using [MJML](https://mjml.io/) markup language.",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
@SarahDonvito ti suggerisco qui di aggiungere uno script tipo "generate" che chiama lo script node generateHtmlTemplate.
da qui possiamo lavorare sui parametri da passare in input allo script node per la generazione dei template